src/Symfony/Component/ErrorHandler/ErrorHandler.php

```diff
@@ -379,7 +374,7 @@ private function reRegister($prev)
      *
      * @internal
      */
-    public function handleError($type, $message, $file, $line)
+    public function handleError(int $type, string $message, string $file, int $line): ?bool
```

Comment:
please remove the param annotations

Reply:
and the nullable return type

Reply:
funny thing is that initially i've removed that block, but afterwards added it again...

Reply:
only `@param` annotations, you can leave the `@return` annotation as its description is relevant in my opinion.